engine/render/src/render/render_script.cpp

```diff
@@ -3203,6 +2870,407 @@ namespace dmRender
     }
 #undef CHECK_COMPUTE_SUPPORT
 
+    /*#
+    * Deprecated in favor of [ref:graphics.STATE_DEPTH_TEST]
+    * @name render.STATE_DEPTH_TEST
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.STATE_STENCIL_TEST]
+    * @name render.STATE_STENCIL_TEST
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.STATE_BLEND]
+    * @name render.STATE_BLEND
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.STATE_CULL_FACE]
+    * @name render.STATE_CULL_FACE
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.STATE_POLYGON_OFFSET_FILL]
+    * @name render.STATE_POLYGON_OFFSET_FILL
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.TEXTURE_FORMAT_LUMINANCE]
+    * @name render.FORMAT_LUMINANCE
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.TEXTURE_FORMAT_RGB]
+    * @name render.FORMAT_RGB
+    * @variable
+    */
+
+    /*#
+    * Deprecated in favor of [ref:graphics.TEXTURE_FORMAT_RGBA]
+    * @name render.FORMAT_RGBA
+    * @variable
```

Comment:
Note: Most of the names are the same as before, but in some occasions the names are different. I've opted for using the more explicit names here that match the actual graphics enum instead of the short "FORMAT_x"